MetaMask/metamask-extension
# fix: Remove old What's New messages

## **Description**

Removes legacy What's New messages, leaving only the Firefox WN *just* so that we can keep the shape of data here.  This PR ensures _no_ What's New messages display at this time.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25048?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2613

## **Manual testing steps**

1. Install fresh MetaMask in Chrome
2. See no "What's New" items.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
